actors/runtime/src/test_utils.rs

```diff
@@ -249,6 +249,10 @@ pub struct ExpectComputeUnsealedSectorCid {
     exit_code: ExitCode,
 }
 
+pub fn expect_ok<T: fmt::Debug>(res: Result<T, ActorError>) -> T {
```

Comment:
This is actors idiomatic sugar for unwrap forcing the result to be ok and returning the result type.  I like this but if this is really un-rusty and you feel strongly that I need to be using unwrap directly let me know.

Reply:
Hmmm. I feel strongly that we should pick one way and be consistent. I'm not sure which way to prefer. 

In either case we should also add this line to the top of every test file to prevent accidental omission of the unwrap:
```
#![deny(unused_must_use)] // Force unwrapping Result<_, Err>
```

Reply:
We should generally be using `unwrap` directly, or using `expect` (even better because we can attach a message).